packages/arb-ts/src/lib/bridge_helpers.ts

```diff
@@ -865,4 +865,8 @@ export class BridgeHelper {
     const whiteList = Whitelist__factory.connect(whiteListAddress, l1Provider)
     return whiteList.isAllowed(address)
   }
+
+  static percentIncrease(num: BigNumber, increase: BigNumber) {
```

Comment:
If we're doing a percent increase of 100%, wouldn't that be `num + num.mul(increase).div(100)`?

Reply:
lel true, it's poorly named. I'll change the logic tho (and keep the name)